manifests/instance.pp

```diff
@@ -18,6 +18,8 @@
 #   enable service (boolean)
 # [*systemd_service_type*]
 #   value for systemd service type
+# [*systemd_workdir*]
```

Comment:
What I don't like about it is that it creates parameters which are usable only by a subset of OS, I don't like distro specific parameters at all. I made an exception for `systemd_service_type` because a hotfix was needed for services that didn't start, but sooner or later I want it to disappear and have a parameter that automatically defaults to the right value depending on the distribution + installation mode. `systemd_workdir` is neither universal nor mandatory for tomcat to operate, therefore I'm not so big on having it as a new parameter.